webapp/app/config.py

```diff
@@ -39,6 +39,10 @@ class BaseConfig(object):
 
     USE_LRU_CACHE = True
 
+    USE_REDIS_LITE = True
+    SESSION_DATA_STORAGE_URL = environ.get('SESSION_DATA_STORAGE_URL')
```

Comment:
Dev environment would work with:
- USE_REDIS_LITE = True
The other variable wouldn't be relevant, since whenever USE_REDIS_LITE = True, the other variable is not needed